src/utils.js

```diff
@@ -28,6 +34,27 @@ export function calculateSlippageAmount(value) {
   return value.mul(SLIPPAGE_PERCENTAGE).div(100)
 }
 
+export async function getUniswapSlippage(
```

Comment:
This looks good, but wondering if we can combine both of the Uniswap utilities into just this one, since `getTradeDetails().executionRate` is the exchange rate we should be using for conversion calculations (our "protection" 95% threshold can just be calculated on top of `executionRate * tokens`).

Reply:
Hmmm I see; makes sense! Even though, the execution rate for small inputs sometimes outputs `0`; it behaves in a strange way.